contracts/src/protocol/OmniAVS.sol

```diff
@@ -127,6 +163,29 @@ contract OmniAVS is IOmniAVS, IOmniAVSAdmin, OwnableUpgradeable, PausableUpgrade
         return address(_avsDirectory);
     }
 
+    /**
+     * @notice Returns the digest hash to be signed by an opeator's validator key on registration.
+     * @param operator      The operator's ethereum address
+     * @param valPubKey     The operator's 64 byte uncompressed secp256k1 validator public key
+     * @param salt          A salt unique to this registration
+     * @param expiry        The timestamp at which this registration expires
+     */
+    function validatorRegistrationDigestHash(address operator, bytes calldata valPubKey, bytes32 salt, uint256 expiry)
```

Comment:
Went with [EIP-712](https://eips.ethereum.org/EIPS/eip-712) compatible signatures, in the same way eigen's [AVSDirectory](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/src/contracts/core/AVSDirectory.sol#L150) does. We may not need "readablity" off chain. But best to follow standards if they're there